examples/esp32c6/runner.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+if [ $# -eq 0 ]; then
+  echo "No arguments supplied! Provide path to ELF as argument"
+fi
+
+outputfilenamecargo=$1
+outputfilename="$outputfilenamecargo".bin
+
+logfile=qemu.log
+
+qemuexec=qemu-system-riscv32
+
+# Building ESP32-C6 image
+espflash save-image --chip esp32c6 --merge "$outputfilenamecargo" "$outputfilename" 1>&2
+
+# Get stats
+esptool.py image_info --version 2 "$outputfilename" 1>&2
+
+# Run in QEMU
+$qemuexec -nographic -monitor tcp:127.0.0.1:55555,server,nowait -icount 3 -machine esp32c6 -drive file="$outputfilename",if=mtd,format=raw -serial file:"$logfile" &
```

Comment:
At present there is no C6 support in [ESP-QEMU](https://github.com/espressif/qemu/releases), and QEMU reports there is no such machine.

We can keep this file, in case they ever add support, but considering it is a copy from C3 it might as well just be copied over if support is added.

Thus I propose you default runner to "the real chip runner", as we won't have QEMU in CI anyhow :)